frontend/src/app/components/comparison/comparison.component.ts

```diff
@@ -0,0 +1,183 @@
+import { Component, OnInit } from '@angular/core';
+import { Company } from 'src/app/models/company-model';
+import { Exchange } from 'src/app/models/exchange-model';
+import { StockPrice } from 'src/app/models/stock-price-model';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/ibmpariwesh/socgen-stockmarket-project/pullRequest?prid=8405405)
```suggestion
import { StockPrice } from "src/app/models/stock-price-model";
```